### PR TITLE
newAssignment

### PR DESCRIPTION
**Pull request by graciousxclusive**
@joshuaunity

### CODE REVIEW

File: assignmment2.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+class BankAccount {
+    constructor(accountNumber, firstName, lastName, dailyWithdrawalLimit) {
+        this.accountNumber = accountNumber;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.accountHolder = `${firstName} ${lastName}`;
+        this.balance = 0;
+        this.transactions = [];
+        this.dailyWithdrawalLimit = dailyWithdrawalLimit;
+        this.withdrawalToday = 0;
```
Comment (joshuaunity) [inline assignmment2.js:18]: Quite creative 🧪

File: assignmment2.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            this.transactions.push(transferTransaction);
+            console.log(`$${amount} transferred to ${recipientAccount.accountHolder}. New balance: $${this.balance}`);
+        } else if (amount <= 0) {
+            console.log("Invalid transfer amount.");
+        } else if (this.balance - amount < 0) {
+            console.log("Insufficient funds.");
+        }
+    }
+
+    viewTransactions() {
```
Comment (joshuaunity) [inline assignmment2.js:62]: This doesnt meet the criteria, you are meant to return an array of JavaScript object/JSON

**Comment by joshuaunity** [comment]
Hi, I'm merging this assignment, but it's important to note that this acknowledgment doesn't imply that all implementations were correct. Only the transfer feature was correctly implemented; the others weren't. I'd appreciate it if you could consistently review your pull requests, as I did mention the issue, but it wasn't addressed.

i know you can do better than this and I look forward to your 4th assignment.

**Review by graciousxclusive — COMMENTED** [review]
@joshuaunity pls review

**Pull request merged**